apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/manager/room/CameraCapturerCompat.kt

```diff
@@ -0,0 +1,135 @@
+package com.twilio.livevideo.app.manager.room
+
+import android.content.Context
+import android.graphics.ImageFormat
+import android.hardware.camera2.CameraCharacteristics
+import android.hardware.camera2.CameraManager
+import android.hardware.camera2.CameraMetadata
+import android.os.Build
+import com.twilio.video.Camera2Capturer
+import com.twilio.video.CameraCapturer
+import com.twilio.video.VideoCapturer
+import timber.log.Timber
+import tvi.webrtc.Camera1Enumerator
+import tvi.webrtc.Camera2Enumerator
+import tvi.webrtc.CameraEnumerator
+import tvi.webrtc.CapturerObserver
+import tvi.webrtc.SurfaceTextureHelper
+
+class CameraCapturerCompat(
+    private val frontCameraId: String?,
+    private val backCameraId: String?,
+    private val cameraCapturer: CameraCapturer? = null,
+    private val camera2Capturer: Camera2Capturer? = null
```

Comment:
What does `camera2Capturer` do?

Reply:
I think, I replied this on the previous comment: https://github.com/twilio/twilio-live-interactive-video/pull/150#discussion_r927078658